### ISSUE TITLE
app-scripts 1.2.0 ios production build fails

### ISSUE THREAD

**Issue by dmastag** [driftyco/ionic-app-scripts#836]
#### Short description of the problem:
After upgrading to ionic-app-scripts 1.2.0 I am getting the following issue

![image](https://cloud.githubusercontent.com/assets/5745805/24317969/c7882222-1130-11e7-9204-a2c856f0f162.png)

Looks like the build has failed

#### What behavior are you expecting?
App build works right like using ionic-app-scripts 1.1.4

**Steps to reproduce:**
1. Update ionic-app-scripts to 1.2.0
2. ionic build ios --prod --release
3. run result in xcode

```
insert any relevant code between the above and below backticks
```

**Which @ionic/app-scripts version are you using?**
1.2.0

**Comment by yasir-netlinks** [comment]
I am having the same problem after updating to `1.2.0`

**Comment by peterpeterparker** [comment]
Looks like both ionic-app-script 1.2.0 and ionic 2.3.0 use typescript 2.2.1 and not 2.0.9 anymore...could that be your error?

**Comment by dmastag** [comment]
Woaahh, did not know that.
Checking the ionic2-app-base package.json and the other starter templates, all are still on 2.0.9.
But ionic-app-scripts indeed has this in the [package.json](https://github.com/driftyco/ionic-app-scripts/blob/master/package.json) file's peerDependencies.

**Comment by Lingomat** [comment]
I have the same issue.

**Comment by dmastag** [comment]
I tried updating typescript to 2.2.1. Same error on production builds (--prod)

**Comment by danbucholtz** [comment]
I am investigating some issues right now. I didn't see any in testing but I'll dive in more and resolve.

Thanks,
Dan

**Comment by danbucholtz** [comment]
Fixed in `1.2.1`. Thanks and sorry for the bug.

Note: If you're using Angular 2.x still requires TS 2.0.x and won't work with TS 2.2.x.

Thanks,
Dan

**Comment by dmastag** [comment]
Confirm that it is working on Android now
